src/components/CreateRules/UploadRules.vue

```diff
@@ -227,6 +264,32 @@ export default class UploadRules extends Mixins(DocumentMixin) {
   /** Called when component is created. */
   private created (): void {
     this.uploadRulesDoc = this.getCreateRulesStep.rulesDoc as File
+    this.rulesConfirmed = this.getCreateRulesStep.rulesConfirmed
+  }
+
+  @Watch('getShowErrors')
+  private onShowErrorsChanged (): void {
+    if (this.getShowErrors && this.$refs.confirmRulesChk) {
+      this.$refs.confirmRulesChk.validate()
+    }
+  }
+
+  @Watch('$route')
+  private async scrollToInvalidComponent (): Promise<void> {
+    if (this.getShowErrors && this.$route.name === RouteNames.CREATE_RULES) {
+      // Scroll to invalid components.
+      await Vue.nextTick()
```

Comment:
Is this required here?

Reply:
not sure what you mean by if it's required here.  are you talking about the `scrollToInvalidComponent` function altogether or `await Vue.nextTick()`?

Reply:
Was referring to the `nextTick()` will resolve though as it's not major.
If it helps smooth things out, i'm all for it :)